projects/packages/publicize/src/class-publicize-script-data.php

```diff
@@ -161,6 +162,41 @@ public static function get_social_settings() {
 		return array(
 			'socialImageGenerator' => $settings->get_image_generator_settings(),
 			'utmSettings'          => $settings->get_utm_settings(),
+			// TODO remove this and use the settings via publicize package.
+			'socialPlugin'         => array(
+				'social_notes_config' => array(),
+			),
+		);
+	}
+
+	/**
+	 * Get the plugin info.
+	 *
+	 * @return array
+	 */
+	public static function get_plugin_info() {
+
+		$social_version  = '0.0.0';
+		$jetpack_version = '0.0.0';
+
+		if ( defined( 'JETPACK_SOCIAL_PLUGIN_ROOT_FILE' ) ) {
+
+			$plugin_data = get_plugin_data( (string) constant( 'JETPACK_SOCIAL_PLUGIN_ROOT_FILE' ), false, false );
+
+			$social_version = $plugin_data['Version'];
+		}
+
+		if ( defined( 'JETPACK__VERSION' ) ) {
+			$jetpack_version = constant( 'JETPACK__VERSION' );
+		}
+
+		return array(
+			'social'  => array(
+				'version' => $social_version,
+			),
+			'jetpack' => array(
+				'version' => $jetpack_version,
+			),
```

Comment:
Do you think we should perhaps set these to null or not include them if the versions are not found, rather than 0.0.0? It could make it easier to check perhaps?

Reply:
Oh yeah, `null` is better. Thanks